master/buildbot/www/hooks/github.py

```diff
@@ -108,6 +130,23 @@ def _get_payload(self, request):
 
         return payload
 
+    def _get_PR_information(self, pr_info):
+        def flatten(properties, base, info_dict):
```

Comment:
I dont like to copy the exact same code in two different places. maybe we should have it in a changes/github.py mixin class